hive/indexer/posts.py

```diff
@@ -168,6 +173,16 @@ def update(cls, op, date, pid):
         if not DbState.is_initial_sync():
             CachedPost.update(op['author'], op['permlink'], pid)
 
+    @classmethod
+    def _get_parent_by_child_id(cls, child_id):
+        """Get parent's `id`, `author`, `permlink` by child id."""
+        sql = """SELECT id, author, permlink FROM hive_posts
```

Comment:
Is the fieldname `permlink` or `permalink`?

Reply:
Internally, `permlink` everywhere